src/main/java/uk/ac/ebi/embl/converter/fftogff3/GFF3AnnotationFactory.java

```diff
@@ -60,26 +61,24 @@ public GFF3Annotation from(Entry entry) throws ValidationException {
 
         String accession = entry.getSequence().getAccession();
         LOG.info("Converting FF entry: {}", accession);
-        // TODO: We need to handle accession versions
-        entry.setPrimaryAccession(accession + ".1");
-        entry.getSequence().setAccession(accession + ".1");
 
-        GFF3Directives directives = new GFF3DirectivesFactory(this.ignoreSpecies).from(entry);
+        GFF3DirectivesFactory directivesFactory = new GFF3DirectivesFactory();
```

Comment:
Instead of creating a new object inject it:
public GFF3AnnotationFactory(boolean ignoreSpecies) {
        this.ignoreSpecies = ignoreSpecies;
    }